dotcom-rendering/src/server/render.article.web.tsx

```diff
@@ -189,7 +193,8 @@ window.twttr = (function(d, s, id) {
 		hasSurveyAd: !!frontendData.config.hasSurveyAd,
 		onlyLightColourScheme:
 			design === ArticleDesign.FullPageInteractive ||
```

Comment:
A quick question(s) for my understanding: are full page interactives not currently built with dark mode support? And is that a current limitation of the platform, or is it just not a design consideration at the moment?

Reply:
Hey @SiAdcock. Sorry for the slow reply, there's been a back and forth going on about this. Interactive articles are built to support dark mode but at present only on apps. On the web light mode is currently forced (see https://github.com/guardian/dotcom-rendering/pull/11982). This isn't a limitation of the platform - more to reduce complexity in an already complex dev progress. That said, now the DCAR migration is finished interactive articles are rendered in the same way across web and apps, so why not align dark mode expectations too?